content/seminars/content.md

```diff
@@ -37,20 +37,34 @@ See you in the next fall semester! -->
 He leads the technical direction of a group ensuring that the network can meet the requirements of availability, reliability and performance.
 He joined Google in 2015 as a network technician working on the Edge, Core and SDN data center network.
 Before Google, Francesco has worked in Cisco as a technical support engineer and has then moved to consulting and finally to an escalation engineering position in a product team in charge of data center L3 switches.
-Francesco holds a MSc in Computer Engineering from Politecnico di Torino. 
+Francesco holds a MSc in Computer Engineering from Politecnico di Torino.
 
 <br>
 <hr>
 
 ### Building and operating the CrownLabs service: architecture and lessons learned
+<!-- ### CrownLabs: A deep-dive journey to unravel the skein -->
 - Speaker: Marco Iorio, [Politecnico di Torino](https://www.polito.it/)
-- When: TO BE SCHEDULED (Jan 2022)
+- When: Tuesday, Jan 11, 2022, 10.00 - 12.00
 
 #### Abstract
-(In progress)
+
+CrownLabs is an open-source project started in March 2020 at Politecnico di Torino to deliver remote computing laboratories.
+Since then, it has been leveraged with satisfaction by the students of more than ten courses, including Cloud Computing, Software Networking, Computer Networks, and Computer Animation, just to cite a few.
+CrownLabs also successfully hosted the September session of the Computer Science (first year course) exams, featuring the integration with the official "exams" platform at PoliTO.
+This seminar presents the CrownLabs architecture and infrastructural setup, mainly addressing the following aspects:
+
+- The motivations and the design of the Kubernetes-powered application back-end;
+- The provisioning of user environments: VMs vs containers;
+- The design and evolution of the CrownLabs frontend;
+- The architectural overview of the Kubernetes-based infrastructure;
+- The lessons learned while operating the service.
 
 #### Short bio
-**Marco Iorio** (in progress).
+
+**Marco Iorio** is a Ph.D student at Politecnico di Torino, where he previously received his M.Sc degree in computer engineering.
+His research interests include multi-cloud systems, edge/fog computing and cyber-security applied to vehicular networks.
+Marco is one of the members of the CrownLabs starting team. Since then, he has been involved in the design and development of the project, serving as tutor of several course projects, as well as he is in charge of operating the CrownLabs infrastructure.
```

Comment:
```suggestion
Marco is one of the founding members of the CrownLabs team. Since then, he has been involved in the design and development of the project, serving as tutor of several course projects, as well as he is in charge of operating the CrownLabs infrastructure.
```